packages/tezos-ts-utils/src/tezos-ts-utils.ts

```diff
@@ -0,0 +1,258 @@
+import { Buffer } from 'buffer';
+const bs58check = require('bs58check');
+
+export { prefix } from './contants';
+
+export function b58cencode(payload: string | Uint8Array, prefix: Uint8Array) {
+  const payloadAr =
+    typeof payload === 'string' ? Uint8Array.from(Buffer.from(payload, 'hex')) : payload;
+
+  const n = new Uint8Array(prefix.length + payloadAr.length);
+  n.set(prefix);
+  n.set(payloadAr, prefix.length);
+
+  return bs58check.encode(Buffer.from(n.buffer));
+}
+
+export const b58cdecode = (enc: string, prefixArg: Uint8Array): Uint8Array =>
+  bs58check.decode(enc).slice(prefixArg.length);
+
+export function b58decode(payload: string) {
```

Comment:
Let's put some time into adding comments to these methods so they don't look so scary to new comers.